doc/tech/approach.md

```diff
@@ -40,7 +40,32 @@ For CNTT, a strategy is needed that affords Operators the performance, flexibili
 > _Placeholder for the Executive Summary (targeting Baldy for first draft). **If you are interested in writing this content, please contact the RM lead.**_
 
 <a name="5.1.2"></a>
-### 5.1.2 Networking Strategy & Principle 
+
+### 5.1.2 Networking Principles & Startegy 
+
+•	Abstraction: A standardized layer between Virtualization Layer domains and the Networking Hardware layer resource pool hiding the implementation details of the Networking Hardware.
```

Comment:
Please use Markdown way of [bulleted list](https://guides.github.com/features/mastering-markdown/).

Reply:
Done , thx